### PR TITLE
[IMP] account: improve invoice/bills view

### PR DESCRIPTION
**Pull request by jbw-odoo**
- Group filters into less technical/more functional approach
- Small tree view change

Task: 2541366

### CODE REVIEW

File: addons/account/views/account_move_views.xml [hunk 1]
```diff
@@ -448,10 +448,10 @@
                     <field name="amount_residual_signed" string="Amount Due" sum="Amount Due" optional="hide"/>
                     <field name="currency_id" groups="base.group_multi_currency" optional="hide"/>
                     <field name="company_currency_id" invisible="1"/>
+                    <field name="to_check" optional="hide" widget="boolean_toggle"/>
+                    <field name="payment_state" widget="badge" decoration-danger="payment_state == 'not_paid'" decoration-warning="payment_state in ('partial', 'in_payment')" decoration-success="payment_state in ('paid', 'reversed')" attrs="{'invisible': [('payment_state', 'in', ('invoicing_legacy'))]}"  optional="show"/>
```
Comment (smetl) [inline addons/account/views/account_move_views.xml:452]: plz indent :p
```
<field name="payment_state"
       widget="badge"
       decoration-danger="payment_state == 'not_paid'"
       decoration-warning="payment_state in ('partial', 'in_payment')"
       decoration-success="payment_state in ('paid', 'reversed')"
       attrs="{'invisible': [('payment_state', 'in', ('invoicing_legacy'))]}"
       optional="show"/>
```

File: addons/account/views/account_move_views.xml [hunk 2]
```diff
@@ -1281,12 +1281,17 @@
...
-                    <filter name="partial" string="Partially Paid" domain="[('payment_state', '=', 'partial'), ('state', '=', 'posted')]"/>
-                    <filter name="paid" string="Paid" domain="[('payment_state', '=', 'paid'), ('state', '=', 'posted')]"/>
-                    <filter name="unpaid" string="Not Paid" domain="[('payment_state', '=', 'not_paid'), ('state', '=', 'posted')]"/>
-                    <filter name="reversed" string="Reversed" domain="[('payment_state', '=', 'reversed'), ('state', '=', 'posted')]"/>
-                    <filter name="late" string="Overdue" domain="['&amp;', ('invoice_date_due', '&lt;', time.strftime('%%Y-%%m-%%d')), ('state', '=', 'posted'), ('payment_state', 'in', ('not_paid', 'partial'))]" help="Overdue invoices, maturity date passed"/>
+                    <!-- in_payment & not_paid &  partial -->
+                    <filter name="open" string="Open" domain="[('state', '=', 'posted'), '|', '|',
+                        ('payment_state', '=', 'in_payment'), ('payment_state', '=', 'not_paid'),
+                        ('payment_state', '=', 'partial')]"/>
```
Comment (smetl) [inline addons/account/views/account_move_views.xml:1287]: `('payment_state', 'in', ('in_payment', 'not_paid', 'partial'))` ?

File: addons/account/views/account_move_views.xml [hunk 3]
```diff
@@ -1281,12 +1281,17 @@
...
+                    <!-- paid & reversed -->
+                    <filter name="closed" string="Closed" domain="[('state', '=', 'posted'), '|',
+                        ('payment_state', '=', 'paid'), ('payment_state', '=', 'reversed')]"/>
```
Comment (smetl) [inline addons/account/views/account_move_views.xml:1290]: `('payment_state', 'in', ('paid', 'reversed'))`

File: addons/account/views/account_move_views.xml [hunk 4]
```diff
@@ -1281,12 +1281,17 @@
...
+                    <!-- overdue -->
+                    <filter name="late" string="Overdue" domain="['&amp;', ('invoice_date_due',
```
Comment (smetl) [inline addons/account/views/account_move_views.xml:1292]: this is confusing: `&` followed by 3 elements

**Review by smetl — CHANGES_REQUESTED** [review]

**Comment by jbw-odoo** [comment]
Done

**Comment by smetl** [comment]
@robodoo r+

**Pull request closed without merging**